scripts/ci/travis-check.js

```diff
@@ -36,14 +37,26 @@ executeCommand( NEW_PACKAGE_DIRECTORY, 'yarn', [ 'run', 'stylelint' ] );
 logProcess( 'Verifying translations...' );
 executeCommand( NEW_PACKAGE_DIRECTORY, 'yarn', [ 'run', 'translations:collect' ] );
 
-logProcess( 'Starting the development server...' );
-startDevelopmentServer( NEW_PACKAGE_DIRECTORY )
-	.then( options => {
-		logProcess( 'Verifying the sample...' );
-		executeCommand( REPOSITORY_DIRECTORY, 'node', [ path.join( 'scripts', 'ci', 'verify-sample.js' ), options.url ] );
-
-		options.server.kill();
+logProcess( 'Creating the DLL build...' );
+executeCommand( NEW_PACKAGE_DIRECTORY, 'yarn', [ 'run', 'dll:build' ] );
```

Comment:
```suggestion
```
This step is unnecessary as the [`prepare`](https://docs.npmjs.com/cli/v8/using-npm/scripts#prepare-and-prepublish) hook builds the DLL files after installing dependencies.